src/ChickenAPI.Game/Entities/Player/Extensions/PlayerInteractionExtensions.cs

```diff
@@ -63,6 +63,11 @@ public static SayPacket GenerateGlobalSayPacket(this IPlayerEntity player, strin
                 Type = color
             };
 
+        public static MsgPacket GenerateHeroPacket(this IPlayerEntity player, string message)
```

Comment:
Extensions should be written inside ChickenAPI.Game.Extensions project and not ChickenAPI.Game